README.md

```diff
@@ -1,2 +1,36 @@
 # django-ilmoitin
-Django app for sending notifications
+
+A templated Django messaging library:
+
+# Installation
+
+1. `pip install django-ilmoitin`
+
+2. Add `django-ilmoitin` to `INSTALLED_APPS`.
+
+# Usage
+
+1. Create a `notifications.py` file in django app.
```

Comment:
Create a `notifications.py` file in your django app and register your notification types:

Reply:
no need for th `# Register event` comment then